src/issueKeyExtended.ts

```diff
@@ -0,0 +1,32 @@
+import aliases from './config/aliases'
+import cliTruncate from 'cli-truncate'
+import chalk from 'chalk'
+
+export enum AliasesPosition {
+    Left,
+    Right
+}
+
+export default async function issueKeyExtended(issueKey: string, aliasesPosition: AliasesPosition) {
+    const aliasNames = await aliases.getAliasNames(issueKey)
+    const issueKeyText = chalk.bold(issueKey)
+    if (aliasNames?.length === 0) return issueKeyText
```

Comment:
Shouldn't this be without a `null` check?

```suggestion
    if (aliasNames.length === 0) return issueKeyText
```